src/utilities/utilities.js

```diff
@@ -40,6 +40,50 @@ export function round(number, precision) {
  * @typedef {"ankle_draft" | "ashrae" | "iso" | "ISO7933"} Standard
  */
 
+/**
+ * Converts degrees to radians unit
+ *
+ * @param {number} degrees
+ *
+ * @returns {number} - radians
+ */
+function degrees_to_radians(degrees) {
+  return degrees * (Math.PI / 180);
+}
+
+/**
+ * Converts radians to degree unit
+ *
+ * @param {number} radians
+ *
+ * @returns {number} - degrees
+ */
+function radians_to_degrees(radians) {
+  return radians * (180 / Math.PI);
+}
+
+/**
+ * Converts sharp and altittude from radians to degree unit
+ * @param {number} sharp
+ * @param {number} altitude
+ * @returns {number}
```

Comment:
this function returns a tuple right? 
```suggestion
 * @returns {[number, number]}
```